kore/src/Kore/Step/TranslateSMT.hs

```diff
@@ -213,16 +214,19 @@ translatePredicate translateUninterpreted predicate =
             { applicationSymbolOrAlias
             , applicationChildren
             }
-      = case getSmtlib (symAttributes smtTools applicationSymbolOrAlias) of
-            Nothing -> empty
+      = case getSmtlib (symAttributes smtlibTools applicationSymbolOrAlias)
+         <|> getSmthook (symAttributes smthookTools applicationSymbolOrAlias)
+         of Nothing -> empty
             Just sExpr -> shortenSExpr <$>
                 applySExpr sExpr
                     <$> zipWithM translatePattern
                         applicationChildrenSorts
                         applicationChildren
         where
```

Comment:
I'm fairly sure there's something wrong about indentation around this line. Since this is a revert of the revert, you can leave it as-is, but you can also fix it.